_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+function add_no_store_header( $headers ) {
+	$headers[ 'Cache-Control' ] .= ', no-store';
```

Comment:
Also, note that other plugins and themes can also modify this, so we might want to make this a bit safer. For example, if the cache control was modified to be `max-age=86400` by another plugin or theme, we probably wouldn't want to add a `no-store` to it.

Reply:
well, if we were going to handle all the possible combinations this would get a little bit complicated. Also, we are changing the cache policy just for jetpack, so it shouldn't affect any other plugin (unless there's a name collusion)

Reply:
> well, if we were going to handle all the possible combinations this would get a little bit complicated.

Fair enough.

> Also, we are changing the cache policy just for jetpack, so it shouldn't affect any other plugin 

Actually, this isn't 100% true - we're changing the cache policy for when we're visiting a Jetpack admin page, but then any link that is visible there (in the masterbar or in the admin sidebar menu) will be affected too. Probably I'm digging too deep about this, but just wanted to make sure it's well thought of.